Tasks/Common/azure-arm-rest/azure-arm-endpoint.ts

```diff
@@ -0,0 +1,118 @@
+import tl = require('vsts-task-lib/task');
+import Q = require('q');
+import webClient = require("./webClient");
+import { AzureEndpoint } from "./azureModels";
+export class AzureRMEndpoint {
+    public _endpoint: AzureEndpoint;
```

Comment:
I assume _variablename format you have choosen for private variables only. At few places public variables are defined with underscore and some places private variables are without underscore. Can we keep pattern same.

Reply:
sure. thanks